### ISSUE TITLE
ICICI bank home page!  radio button is not announced their value

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/bicici#13]
UAT 2022-04
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)

NVDA 2021.3.1
Repro Steps
1. Open the URL  https://www.icicibank.com/
2.press H for heading till to reach to ‘I mobile -pay ’heading level 5and two times tab to reach tab ‘right product for you ‘
3. Reached to the tab ‘find the right product for you ‘
4.press tab  reached to combo box  and select ‘insurance’
5. press tab to reach next combo box and select ‘securing my loved one’
6.press tab and then reached to the radio button
expected Result the radio button should announced their  value
Actual Result the screen reader announced only select radio button one of five and two of five‘
WCAG Guideline
1. https://www.w3.org/TR/WCAG21/#name-role-value
